pypeit/metadata.py

```diff
@@ -893,7 +893,7 @@ def set_configurations(self, configs=None, force=False, fill=None):
                     if uniq_meta.size != 1:
                         msgs.warn('When setting the instrument configuration for {0} '.format(ftype)
                                   + 'frames, configuration {0} does not have unique '.format(cfg_key)
-                                  + '{0} values.' .format(meta))
+                                  + '{0} values.' .format(mkey))
```

Comment:
I'm not sure.  When I ran this on a data set that triggered this statement, the value of `mkey` displayed made sense.  I'm having a little trouble following the logic in this function, but I chose `mkey` since it is a single value inside a `for` loop, and `metakey` would necessarily be a list.

Reply:
Apologies, your right.  It should be `mkey`.  When looking at the file locally, I was looking at the `release` branch instead of `develop`.